### ISSUE TITLE
New Source Requests “Ignis Comic”

### ISSUE THREAD

**Issue by DinilHansana** [keiyoushi/tachiyomi-extensions-source#6891]
### Source name

Ignis Comic

### Source link

https://igniscomic.com/

### Source language

English

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by jamilnielsen** [comment]
according to drakecomic(their discord announcements and comments) they are moving over, that said they have yet to move the backlog of series. (adding a bit of context)

**Comment by Alzack36** [comment]
[Ignis Comic.zip](https://github.com/user-attachments/files/18282251/Ignis.Comic.zip)

I tried making one myself since no one has made it.

*I'm no professional, and this is actually my first time creating a decent extension.

**Comment by Schadows** [comment]
Has someone validated that source apk?

**Comment by Smol-Ame** [comment]
No, anything that isn't directly from our repo shouldn't be trusted/used.

On rare occasions, for testing/troubleshooting purposes, a member (contributor, maintainer) active within the repo would tell you to join & message in Keiyoushi Discord server for extra info & feedback.

**Comment by AwkwardPeak7** [comment]
@Alzack36 please open a pull request instead of sharing apks here
